canopeum_frontend/src/services/api.ts

```diff
@@ -5264,6 +5572,62 @@ export interface IWidget {
   [key: string]: any;
 }
 
+export class Seeds implements ISeeds {
+  id?: number;
+  quantity?: number;
+
+  [key: string]: any;
+
+  constructor(data?: ISeeds) {
+      if (data) {
+          for (var property in data) {
+              if (data.hasOwnProperty(property))
+                  (<any>this)[property] = (<any>data)[property];
+          }
+      }
+  }
+
+  init(_data?: any) {
+      if (_data) {
+          for (var property in _data) {
+              if (_data.hasOwnProperty(property))
+                  this[property] = _data[property];
+          }
+          this.id = _data["id"];
+          this.quantity = _data["quantity"];
+      }
+  }
+
+  static fromJS(data: any): Seeds {
+      data = typeof data === 'object' ? data : {};
+      let result = new Seeds();
+      result.init(data);
+      return result;
+  }
+
+  toJSON(data?: any) {
+      data = typeof data === 'object' ? data : {};
+      for (var property in this) {
+          if (this.hasOwnProperty(property))
+              data[property] = this[property];
+      }
+      data["id"] = this.id;
+      data["quantity"] = this.quantity;
+      return data;
+  }
+
+  toString() {
+    return JSON.stringify(this.toJSON());
+  }
```

Comment:
@NicolasDontigny @BesOlivierBouchard  @bes-alielzein Do you know if this was added manually? I am not able to get it to regenerate with these toString methods.

Possibly related issues:
 - https://github.com/RicoSuter/NSwag/issues/1078
 - https://github.com/RicoSuter/NSwag/issues/1840
 - https://github.com/RicoSuter/NSwag/issues/3163
 - https://github.com/RicoSuter/NSwag/issues/3387